package.json

```diff
@@ -13,11 +13,15 @@
     "private-box": "^0.3.0"
   },
   "devDependencies": {
-    "tape": "^3.0.3"
+    "@types/node": "10.12.9",
+    "tape": "^4.9.0",
+    "dtslint": "0.3.0"
   },
   "scripts": {
-    "test": "tape test/*"
+    "test": "tape test/*",
```

Comment:
make the test script `tape test/*.js && dtslint ...` then it will run both tests at once